p_set_1.rb

```diff
@@ -20,16 +20,22 @@ def ask_minutes
 SPACE =      " "
 BRICKS =     "#"
 APEX_WIDTH = 2
+MIN_HEIGHT = 0
+MAX_HEIGHT = 23
 
 def ask_height
   while true
     puts "height?"
     height = gets.chomp.to_i
-    break if height < 24
+    break if valid_input?(height)
   end
   height
 end
 
+def valid_input?(height)
+  height <= MAX_HEIGHT && height > MIN_HEIGHT
+end
+
```

Comment:
Ah! You can ignore my last comment then. But I think it's a bit confusing for `MIN_HEIGHT` to be `0` when the minimum height is really `1`.